test/distributed/org/apache/cassandra/distributed/impl/InstanceMetrics.java

```diff
@@ -45,16 +45,32 @@
         this.metricsRegistry = metricsRegistry;
     }
 
+    @Override
     public List<String> getNames()
     {
         return new ArrayList<>(metricsRegistry.getNames());
     }
 
+    @Override
     public long getCounter(String name)
     {
-        return metricsRegistry.getCounters().get(name).getCount();
+        Counter counter = metricsRegistry.getCounters().get(name);
```

Comment:
@ifesdjeen can you review this?  every metric has a counter but the current API doesn't allow access to it; my tests need access to a meter's counter